### PR TITLE
docs: update install-dependencies.md doc

### PR DESCRIPTION
**Pull request by Firstsawyou**
### What this PR does / why we need it:

The Mac OSX environment lacks the installation steps of `LuaRocks`. so it needs to be added.
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: doc/install-dependencies.md [hunk 1]
```diff
@@ -150,6 +150,9 @@ nohup etcd &
 # install OpenResty, etcd and some compilation tools
 brew install openresty/brew/openresty etcd luarocks curl git
```
Comment (idbeta) [inline doc/install-dependencies.md:151]: this can be removed
image

Comment (Firstsawyou) [reply doc/install-dependencies.md:151]: To ensure that the installed version of LuaRocks is correct, I think it should be added. This is an error problem encountered when installing according to the document:

截屏2021-02-23 下午7 18 18
截屏2021-02-23 下午7 19 56
截屏2021-02-23 下午7 42 04

Comment (tokers) [reply doc/install-dependencies.md:151]: Since we will install luarocks from source, I think it can be removed.

**Comment by Firstsawyou** [comment]
It has been fixed by https://github.com/apache/apisix/pull/3718.

**Pull request closed without merging**